people-ops/time-off-and-overtime.html.md

```diff
@@ -77,19 +74,42 @@ and so on.
 
 ## Sick leave
 
-For communications regarding sick leave, employees must contact their physician to request a sick 
-leave certificate. The physician will send the certificate electronically to INPS, which, in turn, 
-will make it available in electronic form. Based on the sick leave days granted by the attending 
-physician, the employee must  communicate to the company the days of absence together with the
+We support you in the unfortunate case of sick leave. We pay for sick days a different percentage of
+the daily pay, depending on the event number from the year beginning and the number of days.
+Rates are shown in this table:
+
+| % Pay | Days 1-3 | Days 4-20 | Days 21+ |
+| Event 1 | 100% | 25% | 33% |
+| Event 2 | 100% | 25% | 33% |
+| Event 3 | 100% | 25% | 33% |
+| Event 4 | 100% | 25% | 33% |
+| Event 5+ | 0% | 25% | 33% |
+
+The event counter resets when a new year arrives.
+
+### International employees
+
+To grant your sick leave, we need a medical certificate or medical care proof.
+
+We highly recommend covering the earning loss risk with appropriate insurance coverage.
+
+### Italian employees
+
+For communications regarding sick leave, employees must contact their physician to request a sick
+leave certificate. The physician will send the certificate electronically to INPS, which, in turn,
+will make it available in electronic form. Based on the sick leave days granted by the attending
+physician, the employee must communicate to the company the days of absence together with the
 certificate's unique code.
 
 ## Permit type
```

Comment:
Which permit type should be used for the International employees Holidays?

Reply:
Good question, we have a problem with our current attendance software because it doesn't support multiple holidays' calendars. I'm going to troubleshoot it asap.

Reply:
Thanks 😄